assets/src/dashboard/utils/usePagePreviewSize.js

```diff
@@ -101,23 +101,27 @@ const sizeFromWidth = (
 };
 
 const getContainerWidth = (windowWidth) => {
-  return windowWidth > MIN_DASHBOARD_WIDTH ? windowWidth : MIN_DASHBOARD_WIDTH;
+  // Because the dashboard has a min width (MIN_DASHBOARD_WIDTH) check to see if that min should be used or the actual space of the dashboard
+  const isWindowSmallerThanMinDashboardWidth =
+    window.innerWidth < MIN_DASHBOARD_WIDTH;
+  return isWindowSmallerThanMinDashboardWidth
+    ? MIN_DASHBOARD_WIDTH
+    : windowWidth;
```

Comment:
Looks like a good use case for `window.matchMedia()`

https://developer.mozilla.org/en-US/docs/Web/API/Window/matchMedia

Reply:
TIL! thanks, i'll implement this on Monday!

Reply:
Spent a little time reworking the hook for this and I don't think we should replace our current resize hook with this because limiting to use `.matchMedia` doesn't allow for the dynamic resize we need for larger viewports, it just kicks in for screens smaller than our min width. Perhaps something to circle back to another time, for now I think what we have is ok.

Reply:
ooh yeaup. good call. sounds good 👍